tests/console/validate_lvm_with_raid1.pm

```diff
@@ -0,0 +1,89 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Simple LVM partition validation
+# Maintainer: Yiannis Bonatakis <ybonatakis@suse.com>
+
+use strict;
+use warnings;
+use base "y2_module_consoletest";
+use testapi;
+use utils;
+use Test::Assert ':all';
+use scheduler 'get_test_data';
+
+my $config;
+my @expectedDevs;
+my $expected_num_devs;
```

Comment:
You don't need global variables, you can pass every method what you need. For lvm, you can pass `$config->{lvm}` for example. You might need info from both, but it is not your case. I also would remove $expected_num_devs and would manage internally in the methods. `test_data` represents your expectation and you have two different one for this value, so you should have two different value data in your `test_data`, something like `num_devs` and `num_devs_with_deleted_hdd` or other option it is just subtract one when you do the calculation.

Reply:
i would like to play a bit with it. but i havent done any change as the SLE15 is broken in registration step and i am struggling to test it.